### PR TITLE
Hotfix/cidr (Pi-hole 3.1.1)

### PR DESCRIPTION
**Pull request by PromoFaux**
Let's try this again. 

_Please ensure this is tested thoroughly before approving._

### CODE REVIEW

**Comment by PromoFaux** [comment]
Test results:
https://pastebin.com/raw/rX1uuVKX

File: advanced/Scripts/update.sh [hunk 1]
```diff
@@ -62,7 +66,7 @@ GitCheckUpdateAvail() {
   fi
 
   # Change back to original directory
-  cd "${curdir}"
+  cd "${curdir}" || exit
```
Comment (WaLLy3K) [inline advanced/Scripts/update.sh]: Unless I'm mistaken in its purpose, if `cd` fails, we'll want to use `exit 1` to signify that it quit with an error.

Comment (dschaper) [inline advanced/Scripts/update.sh]: That whole function could use a rewrite, there's no need to store the PWD as the function happens in a subshell so we don't need to move back to the PWD, it's not needed. But that can come in another PR.

**Comment by dschaper** [comment]
`pihole -up` looks okay and `gravity.list` doesn't have any CIDR in it. So I think that covers things?

File: advanced/Scripts/update.sh (hunk 1 above)
Comment (dschaper) [inline advanced/Scripts/update.sh]: I'd fix that now, but that would reset all the current approvals, and it's really not breaking anything it's just extra steps that don't need to be taken.

**Comment by dschaper** [comment]
Approved (since PA doesn't accept `shipit` anymore :( )

**Pull request merged**